Composer/packages/server/src/server.ts

```diff
@@ -164,13 +164,13 @@ export async function start(electronContext?: ElectronContext): Promise<number |
     perMessageDeflate: false,
   });
 
-  const { getLgResources, luImportResolver, staticMemoryResolver } = BotProjectService;
+  const { getLgResources, luImportResolver, staticMemoryResolver, staticEntityResolver } = BotProjectService;
 
   function launchLanguageServer(socket: rpc.IWebSocket) {
     const reader = new rpc.WebSocketMessageReader(socket);
     const writer = new rpc.WebSocketMessageWriter(socket);
     const connection: IConnection = createConnection(reader, writer);
-    const server = new LGServer(connection, getLgResources, staticMemoryResolver);
+    const server = new LGServer(connection, getLgResources, staticMemoryResolver, staticEntityResolver);
```

Comment:
does LG server need an entity resolver?

Reply:
Yes, since LG server needs to suggest Luis entity. We have to provide a way to extract all enitites defined in user's .lu files.